tests/table_test.py

```diff
@@ -69,43 +69,77 @@ def test_calculate_row_column_limits():
         if counter > 0:
-            assert row.size[1] == round(table.row_limits[counter] - table.row_limits[counter-1])
+            assert row.size[1] == round(row_limits[counter] - row_limits[counter-1])
         else:
-            assert row.size[1] == round(table.row_limits[counter])
+            assert row.size[1] == round(row_limits[counter])
+        #Test right color and pixel match
+        assert (row_list[counter].getcolors()[0][1] == dict_items_lst[counter][0]) & (row_list[counter].size == dict_items_lst[counter][1])
         counter+=1
 
 def test_get_cropped_columns():
-    # TODO: Make a test case for get_cropped_columns
     from PIL import Image
-    from table_processing.Table import Table
-
-    file_path = "./tests/resources/simple_table.PNG"
-    table_image = Image.open(file_path).convert("RGB")
-    width, height = table_image.size
-    table_image.resize((int(width*1.75), int(height*1.75)))
-    table = Table(image = table_image)
-    expected_len = (len(table.column_limits))
-    assert  expected_len == len(table.get_cropped_columns(table_image))
+    from table_processing.Table import get_cropped_columns
+    
+    #Creating image
+    column_height = 30
+    column_width = 200
+    im_red = Image.new('RGB', size = (column_width, column_height), color=(255,145,35))
+    im_green = Image.new('RGB', size = (column_width, column_height), color=(125,255,138))
+    im_blue = Image.new('RGB', size = (column_width, column_height), color=(120,150,255))
+    im = Image.new('RGB', size=(im_red.width + im_green.width + im_blue.width, column_height))
+    im.paste(im_red, (0,0))
+    im.paste(im_green, (im_red.width,0))
+    im.paste(im_blue, (im_red.width + im_green.width, 0))
+    
+    #Creating Ditionary holding color and size of each column in image
+    color_dict = {(255,145,35): (column_width, column_height), (125,255,138): (column_width, column_height), (120,150,255): (column_width, column_height)}
+    dict_items_lst = []
+    for item in color_dict.items():
+        dict_items_lst.append(item)
+    
+    #Testing part
+    column_limits = [200,400,600]
+    column_list = get_cropped_columns(im, column_limits)
+    expected_len = len(column_limits)
+    assert  expected_len == len(column_list)
     counter = 0
-    for column in table.get_cropped_columns(table_image):
+    for column in column_list:
+        #Test right dimensions
         if counter > 0:
-            assert column.size[0] == round(table.column_limits[counter] - table.column_limits[counter-1])
+            assert column.size[0] == round(column_limits[counter] - column_limits[counter-1])
         else:
-            assert column.size[0] == round(table.column_limits[counter])
+            assert column.size[0] == round(column_limits[counter])
+        #Test right color and pixel match
+        assert (column_list[counter].getcolors()[0][1] == dict_items_lst[counter][0]) & (column_list[counter].size == dict_items_lst[counter][1])
```

Comment:
Same question: Please confirm that this checks all the pixel values, not just one.